photon-ml/src/integTest/scala/com/linkedin/photon/ml/diagnostics/fitting/FittingDiagnosticIntegTest.scala

```diff
@@ -23,22 +23,24 @@ import com.linkedin.photon.ml.supervised.model.GeneralizedLinearModel
 import com.linkedin.photon.ml.test.SparkTestUtils
 import org.apache.spark.rdd.RDD
 import org.testng.annotations.Test
+import org.testng.Assert._
 
 /**
  * Integration tests for FittingDiagnostic
  */
 class FittingDiagnosticIntegTest extends SparkTestUtils {
+
   import FittingDiagnosticIntegTest._
-  import org.testng.Assert._
 
   /**
    * Happy path
    */
   @Test
   def checkHappyPath():Unit = sparkTest("checkHappyPath") {
-    val data = sc.parallelize(drawBalancedSampleFromNumericallyBenignDenseFeaturesForBinaryClassifierLocal(SEED, SIZE, DIMENSION).map( x => new LabeledPoint(x._1, x._2)).toSeq).repartition(4).cache
+    val data = sc.parallelize(drawBalancedSampleFromNumericallyBenignDenseFeaturesForBinaryClassifierLocal(SEED, SIZE,
+      DIMENSION).map( x => new LabeledPoint(x._1, x._2)).toSeq).repartition(NUM_PARTITIONS).cache()
```

Comment:
I think the correct Scala style for this would be:
```
val data = sc.parallelize(
  drawBalancedSampleFromNumericallyBenignDenseFeaturesForBinaryClassifierLocal(SEED, SIZE, DIMENSION)
    .map( x => new LabeledPoint(x._1, x._2) )
    .toSeq)
  .repartition(NUM_PARTITIONS)
  .cache()
```